rust/private/clippy.bzl

```diff
@@ -72,7 +72,7 @@ def _get_clippy_ready_crate_info(target, aspect_ctx):
 
 def _clippy_aspect_impl(target, ctx):
     crate_info = _get_clippy_ready_crate_info(target, ctx)
-    if not crate_info:
+    if not crate_info or "__bindgen" in ctx.rule.attr.tags:
```

Comment:
instead of checking this here, could we instead add `noclippy` tags to the `bindgen` targets? This way the clippy rules don't need to be aware of bindgen specifically.

Reply:
Done.